test/unit/sanitiser/_sources_and_layers.js

```diff
@@ -53,7 +53,7 @@ module.exports.tests.no_errors = function(test, common) {
 
 test('valid combination', function(t) {
     var raw = {};
-    var clean = { sources: ['geonames'], layers: ['macroregion'] };
+    var clean = { sources: ['geonames'], layers: ['borough'] };
```

Comment:
please duplicate this test, keeping the macroregion one, and adding a new borough one :P